101-azure-api-management-create/azuredeploy.json

```diff
@@ -1,80 +1,60 @@
-	    }
-        }
-        ,
-        "publisherName":
-        {
-            "type": "string",
-            "minLength": 1,
-	    "metadata": {
-		"description": "The name of the owner of the service"
-	    }	
-        }
-        ,
-        "sku":
-        {
-            "type": "string",
-            "allowedValues": [
-            "Developer",
-            "Standard",
-            "Premium"
-            ],
-            "defaultValue": "Developer",
-	    "metadata": {
-		"description": "The pricing tier of this API Management service"
-	    }	
-        }
-        ,
-        "skuCount":
-        {
-            "type": "string",
-            "allowedValues": [
-            "1",
-            "2"
-            ],
-            "defaultValue": "1",
-	    "metadata": {
-		"description": "The instance size of this API Management service."
-	    }	
-        }
-    }
-    ,
-    "variables":
-    {
-        "apiManagementServiceName": "[concat('apiservice', uniqueString(resourceGroup().id))]"
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "publisherEmail": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "The email address of the owner of the service"
+      }
+    },
+    "publisherName": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "The name of the owner of the service"
+      }
+    },
+    "sku": {
+      "type": "string",
+      "allowedValues": [
+        "Developer",
+        "Standard",
+        "Premium"
+      ],
+      "defaultValue": "Standard",
+      "metadata": {
+        "description": "The pricing tier of this API Management service. Developer is not recommended for production and has no SLA"
+      }
+    },
+    "capacity": {
+      "type": "int",
+      "defaultValue": 1,
+      "metadata": {
+        "description": "The number of units this API Management service has. The performance per unit varies based on tier. See the FAQ here for more information: https://azure.microsoft.com/en-us/pricing/details/api-management/"
+      }
     }
-    ,
-    "resources": [
+  },
+  "variables": {
+    "apiManagementServiceName": "[concat('apiservice', uniqueString(resourceGroup().id))]"
+  },
+  "resources": [
     {
-        "apiVersion": "2014-02-14",
-        "name": "[variables('apiManagementServiceName')]",
-        "type": "Microsoft.ApiManagement/service",
-        "location": "West US",
-        "tags":
-        {
-        }
-        ,
-        "properties":
-        {
-            "sku":
-            {
-                "name": "[parameters('sku')]",
-                "capacity": "[parameters('skuCount')]"
-            }
-            ,                
-            "publisherEmail": "[parameters('publisherEmail')]",
-            "publisherName": "[parameters('publisherName')]"
-        }
+      "apiVersion": "2016-07-07",
```

Comment:
Hey @daschult I have not, I essentially used this method:
https://azure.microsoft.com/en-us/documentation/articles/resource-manager-supported-services/#supported-api-versions
to find the latest API version supported by the resource then discovered the changes required in the template via some trial and error.

I just took a look at the schemas repo, but there doesn't appear to be too much info in the way of how to contribute new schema. I could take a crack at it going by what I figured out, but I am not sure what the 'correct' way to do it is. If you could provide some advice on that (perhaps out of this thread, or point me to some resources, a mailing list etc.) I'd be happy to try though.